docker/pypi/wmagent/install.sh

```diff
@@ -48,11 +48,22 @@ done
 WMA_TAG_REG="^[0-9]+\.[0-9]+\.[0-9]{1,2}((\.|rc)[0-9]{1,2})?$"
 [[ $WMA_TAG =~ $WMA_TAG_REG ]] || { echo "WMA_TAG: $WMA_TAG does not match requered expression: $WMA_TAG_REG"; echo "EXIT with Error 1"  ; exit 1 ;}
 
+# Parsing the WMA_TAG
+declare -A WMA_VER
+WMA_VER[release]=$(echo $WMA_TAG|sed -nr 's/.*(^[0-9]+\.[0-9]+\.[0-9]+).*$/\1/p')
+WMA_VER[patch]=$(echo $WMA_TAG|sed -nr 's/.*(^[0-9]+\.[0-9]+\.[0-9]+)//p')
+WMA_VER[major]=$(echo $WMA_TAG|sed -nr 's/.*(^[0-9]+\.[0-9]+).*/\1/p')
+WMA_VER[minor]=$(echo $WMA_TAG|sed -nr 's/^[0-9]+\.[0-9]+\.([0-9]+).*$/\1/p')
+
 echo
 echo "======================================================================="
 echo "Starting new WMAgent deployment with the following initialisation data:"
 echo "-----------------------------------------------------------------------"
-echo " - WMAgent Version            : $WMA_TAG"
+echo " - WMAgent Tag                : $WMA_TAG"
```

Comment:
I would suggest to keep it saying "WMAgent Version".
Then rephrase "WMAgent Release" to "WMAgent Release Cycle".
Lastly, remove the Major, Minor and Patch, and those might be confusing and maybe even incorrect (e.g., for release 2.3.3).

Reply:
ok, 

... the way I dd it was to be extra explicit in the meaning and the matching between the print outs and the variables themselves, but that's minor. What I cannot understand is why would the `major/minor` and `patch`  part of the tag be incorrect for a tag like `WMA_TAG=2.3.3` ? I have tested it many times  and in the case of a tag like this what those pirntouts would say would be:
```
#11 0.115 =======================================================================
#11 0.115 Starting new WMAgent deployment with the following initialisation data:
#11 0.115 -----------------------------------------------------------------------
#11 0.116  - WMAgent Tag                : 2.3.3
#11 0.116  - WMAgent Release            : 2.3.3
#11 0.116  - WMAgent Major version      : 2.3
#11 0.116  - WMAgent Minor version      : 3
#11 0.116  - WMAgent Patch/Release cand : 
#11 0.116  - WMAgent User               : 
#11 0.116  - WMAgent Root path          : /data
#11 0.118  - Python  Version            : Python 3.8.16
#11 0.118  - Python  Module path        : /usr/local/lib/python3.8/site-packages
#11 0.118 =======================================================================
``` 

Which is exactly what is expected I believe.

Reply:
I think the concept of release type for our tagging convention is not straight forward. For instance, for 2.3.4, I would say:
* major release: 2
* middle release: 3
* minor release: 4

but for 2.3.4.5, I see two ways to read it. Either as:
* major release: 2
* middle release: 3
* minor release: 4
* patch release: 5
OR
* major release: 2
* middle release: 3.4
* minor release: 5
* patch release: 

Due to the potential confusion on this interpretation, I feel like it is better not to give such information.